cmd/csi-resizer/main.go

```diff
@@ -70,7 +76,21 @@ func main() {
 	}
 	klog.Infof("Version : %s", version)
 
-	kubeClient, err := util.NewK8sClient(*master, *kubeConfig)
+	var config *rest.Config
+	var err error
+	if *master != "" || *kubeConfig != "" {
+		config, err = clientcmd.BuildConfigFromFlags(*master, *kubeConfig)
+	} else {
+		config, err = rest.InClusterConfig()
+	}
+	if err != nil {
+		klog.Fatal(err.Error())
+	}
+
+	config.QPS = float32(*kubeAPIQPS)
+	config.Burst = *kubeAPIBurst
+
+	kubeClient, err := kubernetes.NewForConfig(config)
```

Comment:
Can you also create a new clientset for leader election like what you did in external-provisioner?

Reply:
done!